src/main/java/de/unistuttgart/informatik/fius/icge/workbench/swing/SwingView.java

```diff
@@ -196,7 +205,48 @@ private void initLogPanel(JPanel main) {
         JButton clearLogButton = new JButton("clear log");
         clearLogButton.addActionListener(e -> this.clearLog());
         logPanel.add(BorderLayout.SOUTH, clearLogButton);
-        
-        main.add(BorderLayout.EAST, logPanel);
+
+        JPanel subPanel = new JPanel(new BorderLayout());
+        subPanel.add(logPanel);
+
+        subPanel.add(BorderLayout.PAGE_END, initSpeedSlider());
```

Comment:
I think it would be more appropriate to add the speed slider to the `ToolBar` instead.